data/net/minecraft/server/players/IpBanListEntry.mapping

```diff
@@ -1,4 +1,10 @@
 CLASS net/minecraft/server/players/IpBanListEntry
+	METHOD <init> (Lcom/google/gson/JsonObject;)V
+		ARG 1 entryData
+	METHOD <init> (Ljava/lang/String;)V
+		ARG 1 ip
+	METHOD <init> (Ljava/lang/String;Ljava/util/Date;Ljava/lang/String;Ljava/util/Date;Ljava/lang/String;)V
+		ARG 1 ip
```

Comment:
Why wasn't the other parameters of this constructor named, given that they are available from looking at the superclass constructor which it calls?